.circleci/config.yml

```diff
@@ -250,6 +250,11 @@ defaults:
       requires:
         - b_ubu_ossfuzz
 
+  - workflow_win: &workflow_win
+      <<: *workflow_trigger_on_tags
+      requires:
+        - b_win
```

Comment:
``b_win_release``?

Reply:
Yep, once https://github.com/ethereum/solidity/pull/9476 is done. I'll rebase this one once the current runs went through, otherwise I have to wait for 20 more minutes to see if the test results uploaded correctly.